docs/operators_and_staking/operators.md

```diff
@@ -0,0 +1,261 @@
+-- \
+--domain-id 0 \
+--chain local \
+--operator \
+--keystore-path /tmp/keystore \
+--rpc-external
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+The `node-key` in the example above can by any `hex` string of the same length.
+:::
+
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+
+For `dev` chain, you can use `--dev` flag that combines `--chain dev` and `--operator`.
+
+```powershell
+target/production/subspace-node `
+--dev `
+--rpc-external `
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 `
+-- `
+--domain-id 0 `
+--dev `
+--rpc-external
+```
+
+
+Run another node and sync the consensus chain from the consensus node started earlier:
+
+```powershell
+target/production/subspace-node `
+--dev `
+--rpc-external `
+--bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp `
+-- `
+--domain-id 0 `
+--chain dev `
+--rpc-external
+```
+
+</TabItem>
+
+
+<TabItem value="macos" label="🍎 macOS">
+
+For `dev` chain, you can use `--dev` flag that combines `--chain dev` and `--operator`.
+```bash
+target/production/subspace-node \
+--dev \
+--rpc-external \
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+-- \
+--domain-id 0 \
+--dev \
+--rpc-external
+```
+
+Run another node and sync the consensus chain from the consensus node started earlier:
+
+```bash
+target/production/subspace-node \
+--dev \
+--rpc-external \
+--bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp \
+-- \
+--domain-id 0 \
+--chain dev \
+--rpc-external
+```
+
+</TabItem>
+
+
+<TabItem value="linux" label="🐧 Ubuntu">
+
+
+For `dev` chain, you can use `--dev` flag that combines `--chain dev` and `--operator`.
+
+
+```bash
+target/production/subspace-node \
+--dev \
+--rpc-external \
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+-- \
+--domain-id 0 \
+--dev \
+--rpc-external
+```
+
+Run another node and sync the consensus chain from the consensus node started earlier:
+
+```bash
+target/production/subspace-node \
+--dev \
+--rpc-external \
+--bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp \
+-- \
+--domain-id 0 \
+--chain dev \
+--rpc-external
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Specified bootnode ID should always correspond to the `node key`, in the example above, bootnode ID `12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp` corresponds to the `node key` `0000000000000000000000000000000000000000000000000000000000000001` 
+:::
+
+Since there is no `operator` flag, this node will not participate in Bundle production.
+By default, node data is written to `subspace-node/domain-{domain-id}` subdirectory of the OS-specific user's local data directory.
+
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+FOLDERID_LocalAppData C:\Users\Alice\AppData\Local
```

Comment:
These folders and their examples are a little difficult to separate when rendered. Consider something like:
FOLDERID_LocalAppData e.g. C:\Users\Alice\AppData\Local
or maybe bracketed or italicised?